messages/en.json

```diff
@@ -7,14 +7,15 @@
   "admin/editor.store-image.title.description": "Title to be shown on hover",
   "admin/editor.image-list.title": "Image List",
   "admin/editor.image-list.description": "Show a list of images",
+  "admin/editor.image-list.images.title": "Images",
   "admin/editor.image-list.images.image.title": "Image",
   "admin/editor.image-list.images.mobileImage.title": "Image for mobile",
-  "admin/editor.image-list.images.description.title": "Alt image description",
+  "admin/editor.image-list.images.description.title": "Image alt text",
+  "admin/editor.image-list.images.title.title": "Image title",
   "admin/editor.image-list.images.link.url.title": "Image link URL",
   "admin/editor.image-list.images.link.openNewTab.title": "Should open on new tab",
-  "admin/editor.image-list.images.link.noFollow.title": "Shuold be a nofollow link",
-  "admin/editor.image-list.images.link.title.title": "Title for the image",
-  "admin/editor.image-list.images.title": "Images",
+  "admin/editor.image-list.images.link.noFollow.title": "Should be a nofollow link",
+  "admin/editor.image-list.images.link.title.title": "Title for the link",
```

Comment:
This is not being used, is it?

Reply:
I think not but I don't have 100% sure. Do you think we can remove that?

Reply:
@klzns

Reply:
If there's no reference to this string in this project, yes